src/lib/modules/Document/DocumentAuthors.js

```diff
@@ -195,15 +197,21 @@ class DocumentAuthors extends Component {
       displayedAuthors.length > scrollLimit && isExpanded ? 'expanded' : '';
     return (
       <Overridable id="DocumentAuthors.layout" {...this.props}>
-        <div className={`document-authors-list-wrapper ${scrollableClass}`}>
+        <div
+          className={
+            noClassName
+              ? `${scrollableClass}`
+              : `document-authors-list-wrapper ${scrollableClass}`
+          }
+        >
           {prefix}
           <List horizontal className="document-authors-list" as={listItemAs}>
             {displayedAuthors.map((author, index) => {
               const isLast = index === displayedAuthors.length - 1;
               return (
                 <List.Item key={author.full_name}>
                   {author.full_name}
-                  {withPopUpShowMoreFields && (
+                  {withPopUpShowMoreFields && allAuthors.length < 5 && (
```

Comment:
Just for the sake of naming numbers in the code, I would have this as a prop:
```suggestion
                  {withPopUpShowMoreFields && allAuthors.length < showMaxAuthorsCount && (
```
and then
```
DocumentAuthors.defaultProps = {
    ...
    showMaxAuthorsCount: 5
```
What do you think?

Reply:
Yes, thanks for noticing! We can use `invenioConfig.LITERATURE.authors.maxDisplay` instead